src/library/impl/meta_reader/schema.h

```diff
@@ -61,6 +61,11 @@ namespace xlang::meta::reader
             return get_string(2);
         }
 
+        auto FullyQualifiedName() const
```

Comment:
It seems like a mistake to add this to meta reader. First, you don't have a delimiter between NS and name. Second, different languages use different symbols to delimit - dot for C# vs double colon for C++ for example.